### ISSUE TITLE
Ordering by 'abr' orders alphabetically, not numerically

### ISSUE THREAD

**Issue by ndg63276** [pytube/pytube#252]
Ordering by 'abr', gives orders as if sorted alphabetically, so 128 < 160 < 50 < 70, eg

`>>> yt=YouTube('https://www.youtube.com/watch?v=9bZkp7q19f0')`
`>>> yt.streams.filter(only_audio=True).order_by('abr').all()`
`[<Stream: itag="140" mime_type="audio/mp4" abr="128kbps" acodec="mp4a.40.2">, <Stream: itag="171" mime_type="audio/webm" abr="128kbps" acodec="vorbis">, <Stream: itag="251" mime_type="audio/webm" abr="160kbps" acodec="opus">, <Stream: itag="249" mime_type="audio/webm" abr="50kbps" acodec="opus">, <Stream: itag="250" mime_type="audio/webm" abr="70kbps" acodec="opus">]`

Is it possible to do a natural sort, so that 50 < 70 < 128 < 160?

**Comment by Mattwmaster58** [comment]
formatted
```
>>> yt=YouTube('https://www.youtube.com/watch?v=9bZkp7q19f0')
>>> yt.streams.filter(only_audio=True).order_by('abr').all()
[<Stream: itag="140" mime_type="audio/mp4" abr="128kbps" acodec="mp4a.40.2">, 
 <Stream: itag="171" mime_type="audio/webm" abr="128kbps" acodec="vorbis">, 
 <Stream: itag="251" mime_type="audio/webm" abr="160kbps" acodec="opus">,
 <Stream: itag="249" mime_type="audio/webm" abr="50kbps" acodec="opus">,
 <Stream: itag="250" mime_type="audio/webm" abr="70kbps" acodec="opus">]
```
The `order_by` just runs everything through the `sorted` function with `key` set to `lambda thingtosortby: getattr(self, thingtosortby)`.

**Comment by Mattwmaster58** [comment]
Related to #194

**Comment by vn-ki** [comment]
Closed in #287
